css-styleguide.md

```diff
@@ -0,0 +1,652 @@
+# Coding styleguide
+
+1. Preproocessor
+2. Frameworks
+3. Syntax & Formatting
+4. Units
+5. Naming
+6. Variables
+7. Specificity
+8. Architecture
+9. Documentation
+
+### Things to cover
+- Being careful with @extend -> architecture
+- Breakpoints
+- Sass output and folder structure there -> architecture
+- Mixins and functions
+- Full example
+
+## Preprocessor
+The  most supported preprocessor is sass/scss. Using this pre-processor means you'll get supported resources such as frameworks, libraries, tutorials and a comprehensive styleguide as support.
```

Comment:
Do you mean the most supported preprocessor at 18F or in general? Agree that this the most supported one at 18F, if that's what you mean, and that this is a good way to suggest that people consider using it ("this means you'll get XYZ").